packages/tools/generator-widget/generators/app/lib/prompttexts.js

```diff
@@ -64,7 +64,7 @@ function promptWidgetProperties(mxProjectDir, widgetName) {
             type: "input",
             name: "projectPath",
             message: "Mendix project path",
-            default: mxProjectDir ? mxProjectDir : "./dist/MxTestProject"
+            default: mxProjectDir ? mxProjectDir : "./tests/TestProject"
```

Comment:
can you make it `testProject`? Otherwise it conflicts with !276

Reply:
Changed it.